config/initializers/mock_saml.rb

```diff
@@ -4,7 +4,9 @@
   users: [
     OpenStruct.new(username: 'test1', email: 'test1@example.com'),
     OpenStruct.new(username: 'test2', email: 'test2@example.com'),
-    OpenStruct.new(username: 'test3', email: 'really-really-long-email-address@example.com')
+    OpenStruct.new(username: 'test3', email: 'really-really-long-email-address@example.com'),
+    OpenStruct.new(username: 'NEETADESOR', email: 'neeta@desor.com'),
+    OpenStruct.new(username: 'DAVIDGRAYLLPTWO', email: 'dg@dgllp.com')
```

Comment:
These look like real email addresses. Can we change them to something fake to prevent emails being sent out accidentally to them? Something like `neeta@example.com` and `dg@example.com`